pepsico/projections/maproom.py

```diff
@@ -322,6 +322,34 @@ def write_map_title(
         )
 
 
+    APP.clientside_callback(
+        """function(start_year, end_year, start_year_ref, end_year_ref) {
+            if (!(!start_year || !end_year)) {
```

Comment:
The double negation is a bit confusing. Can't we rewrite this as follows?
```
if (start_year && end_year)
```
Similarly in line 331.

Also confusing is assigning a new value (especially of a different type) to a parameter. I would prefer to introduce new variables `valid_start` etc. instead of changing the value of `start_year` etc.